### ISSUE TITLE
Add more kernel config checks

### ISSUE THREAD

**Issue by v-thakkar** [parca-dev/parca-agent#687]
This is for the follow-up PR after #686 has been merged.
 - [ ] Check for the stack unwinding-related options
 - [ ] Check for the CONFIG_SYS_ADMIN, CAP_PERF_MON (for perf_events)
 - [ ] Check for Cgroup versions

**Comment by kakkoyun** [comment]
For capability checks: https://github.com/aquasecurity/tracee/tree/main/pkg/capabilities

**Comment by palash25** [comment]
i would like to take this up if no one is working on this
